unstructured/ingest/connector/wikipedia.py

```diff
@@ -22,22 +22,30 @@
 @dataclass
 class SimpleWikipediaConfig(BaseConnectorConfig):
     title: str
-    auto_suggest: bool
+    auto_suggest: bool = False
 
 
 @dataclass
 class WikipediaIngestDoc(IngestDocCleanupMixin, BaseIngestDoc):
     connector_config: SimpleWikipediaConfig = field(repr=False)
+    revision_id: t.Optional[int] = None
```

Comment:
actually, now that I'm looking at this, revision_id is already assigned to the version via source metadata. do we need to store as revision_id on the instance directly as well?